lib/src/map/camera/camera.dart

```diff
@@ -249,87 +248,89 @@ class MapCamera {
 
   /// Calculates the pixel bounds of this camera's [crs].
-  Bounds? getPixelWorldBounds(double? zoom) =>
+  Rect? getPixelWorldBounds(double? zoom) =>
       crs.getProjectedBounds(zoom ?? this.zoom);
 
   /// Calculates the [Offset] from the [pos] to this camera's [pixelOrigin].
-  Offset getOffsetFromOrigin(LatLng pos) =>
-      (project(pos) - pixelOrigin).toOffset();
+  Offset getOffsetFromOrigin(LatLng pos) => project(pos) - pixelOrigin;
 
   /// Calculates the pixel origin of this [MapCamera] at the given
   /// [center]/[zoom].
-  Point<int> getNewPixelOrigin(LatLng center, [double? zoom]) {
-    final halfSize = size / 2.0;
-    return (project(center, zoom) - halfSize).round();
+  Offset getNewPixelOrigin(LatLng center, [double? zoom]) {
+    return (project(center, zoom) - size.center(Offset.zero)).round();
   }
 
   /// Calculates the pixel bounds of this [MapCamera]. This value is cached.
-  Bounds<double> get pixelBounds =>
+  Rect get pixelBounds =>
       _pixelBounds ?? (_pixelBounds = pixelBoundsAtZoom(zoom));
 
   /// Calculates the pixel bounds of this [MapCamera] at the given [zoom].
-  Bounds<double> pixelBoundsAtZoom(double zoom) {
-    var halfSize = size / 2;
+  Rect pixelBoundsAtZoom(double zoom) {
+    Size cameraSize = size;
     if (zoom != this.zoom) {
       final scale = getZoomScale(this.zoom, zoom);
-      halfSize = size / (scale * 2);
+      cameraSize = size / (scale * 2);
     }
-    final pixelCenter = project(center, zoom).floor().toDoublePoint();
-    return Bounds(pixelCenter - halfSize, pixelCenter + halfSize);
+    final pixelCenter = project(center, zoom).floor();
+
+    return Rect.fromCenter(
+        center: pixelCenter,
+        width: cameraSize.width,
+        height: cameraSize.height);
   }
 
   /// This will convert a latLng to a position that we could use with a widget
   /// outside of FlutterMap layer space. Eg using a Positioned Widget.
-  Point<double> latLngToScreenPoint(LatLng latLng) {
-    final nonRotatedPixelOrigin = project(center, zoom) - nonRotatedSize / 2.0;
+  Offset latLngToScreenOffset(LatLng latLng) {
+    final nonRotatedPixelOrigin =
+        project(center, zoom) - nonRotatedSize.center(Offset.zero);
 
-    var point = crs.latLngToPoint(latLng, zoom);
+    var point = crs.latLngToOffset(latLng, zoom);
 
-    final mapCenter = crs.latLngToPoint(center, zoom);
+    final mapCenter = crs.latLngToOffset(center, zoom);
 
     if (rotation != 0.0) {
-      point = rotatePoint(mapCenter, point, counterRotation: false);
+      point = rotateOffset(mapCenter, point, counterRotation: false);
     }
 
     return point - nonRotatedPixelOrigin;
   }
 
   /// Calculate the [LatLng] coordinates for a [localPoint].
-  LatLng pointToLatLng(Point localPoint) {
-    final localPointCenterDistance = Point(
-      (nonRotatedSize.x / 2) - localPoint.x,
-      (nonRotatedSize.y / 2) - localPoint.y,
+  LatLng offsetToLatLng(Offset localPoint) {
```

Comment:
Should this be called `screenOffsetToLatLng` for consitency, or are they different?